### PR TITLE
MS SQL Server Sink Kamelet

### LINKED ISSUE #364: Create MS SQL Server Sink Kamelet

### PR DESCRIPTION
**Pull request by oscerd**
Fixes #364

### CODE REVIEW

File: sqlserver-sink.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    camel.apache.org/kamelet.group: "SQL"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Microsoft SQL Server Sink"
+    description: |-
+      Send data to a Microsoft SQL Server Database.
+
+      In you KameletBinding file you'll need to explicitly declare the MariaDB driver dependency, like in the following YAML snippet
```
Comment (nicolaferraro) [inline sqlserver-sink.kamelet.yaml:19]: Typo about maria DB. Think we can reference the driver directly here

Comment (nicolaferraro) [reply sqlserver-sink.kamelet.yaml:19]: There are also other refs to mariaDB

**Review by nicolaferraro — CHANGES_REQUESTED** [review]
I think we should put here and in the other SQL connectors some instructions. e.g. specifying that the input is expected to be JSON (in spec->types), an example in the query field and telling how the json fields are mapped to parameter values

**Comment by oscerd** [comment]
I'll rework the other connector by adding expecting type, query example and json mapping.

**Comment by oscerd** [comment]
Let me merge and open other PRs.

**Pull request merged**